src/Native/pkg/runtime.native.System.IO.Compression/win/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -6,11 +6,19 @@
     <WinVersion Condition="'$(PackagePlatform)'=='arm64'">win10</WinVersion>
     <WinVersion Condition="'$(PackagePlatform)'=='x86' OR '$(PackagePlatform)'=='x64'">win7</WinVersion>
     <PackageTargetRuntime>$(WinVersion)-$(PackagePlatform)</PackageTargetRuntime>
+    <PackageVersion>4.3.1</PackageVersion>
+    <!-- the native compression packages require nuget >3.5 for nativeassets support -->
+    <MinClientVersion3>3.5</MinClientVersion3>    
   </PropertyGroup>
   <ItemGroup>
     <NativeFile Include="$(WinNativePath)\clrcompression.dll">
       <TargetPath>runtimes/$(WinVersion)-$(PackagePlatform)/native</TargetPath>
     </NativeFile>
+    <!-- We redistribute the 4.0.1 binaries for netcore50 because they target the API sets 
+         as kernel32 is unavailable on OneCore systems -->
+    <ProjectReference Condition="'$(PackagePlatform)'!='arm64' AND !Exists('$(WinNativePath)..\clrcompression\clrcompression.dll')" Include="redist\clrcompression.depproj">
```

Comment:
I don't think you want this Exists check. It will cause this package to have different dependencies based on file system state.